packages/discovery-provider/src/api/v1/models/tracks.py

```diff
@@ -86,7 +92,7 @@
         "orig_filename": fields.String(
             allow_null=True
         ),  # remove nullability after backfill
-        "is_original_available": fields.Boolean,
+        "is_original_available": fields.Boolean(),
```

Comment:
What's the difference here between calling it and passing a reference to it?

Reply:
Nothing, afaik - I think this is just a random remnant from testing something